app/models/card.rb

```diff
@@ -1,6 +1,10 @@
 require 'super_memo'
+require 'translationable'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.